model_training/script.sh

```diff
@@ -180,7 +181,7 @@ function quick_collect() {
 }
 
 function train() {
-    train_model stressng_kepler_query,coremark_kepler_query,parsec_kepler_query ${VERSION}
+    train_model stressng_kepler_query ${VERSION}
```

Comment:
Is this selecting the data only for training?

If it is also for the validation: should we also use the coremark results for testing to verify the accuracy of the model with different workload?

Reply:
It will be shuffled and use 10% of it for validation.
https://github.com/sustainable-computing-io/kepler-model-server/blob/6b4f716764dcf2ad7b1dc0371554c3acd8d7457b/src/train/trainer/__init__.py#L28

We need to refactor the code to have a fixed validation dataset. Should be created for a separate issue.